mne/channels/layout.py

```diff
@@ -907,19 +907,23 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         raise ValueError("'unit' should be either 'm', 'cm', or 'mm'.")
     if transform is True:
         names_lower = [name.lower() for name in list(ch_names_)]
-        fids = ('nasion', 'lpa', 'rpa')
-        # check that all needed points are present
+        if ext == '.hpts':
+            fids = ('2', '1', '3')  # Alternate cardinal point names
+        else:
+            fids = ('nasion', 'lpa', 'rpa')
+
         missing = [name for name in fids
                    if name not in names_lower]
         if missing:
             raise ValueError("The points %s are missing, but are needed "
-                             "to transform the points to the MNE coordinate "
-                             "system. Either add the points, or read the "
-                             "montage with scale as bool." % missing)
+                             "to transform the points to the MNE "
+                             "coordinate system. Either add the points, "
+                             "or read the montage with scale as bool."
```

Comment:
Sorry I did not notice before, this error message seems outdated (not your fault @haribharadwaj ) shouldn't it say "use with `transform=False`"?